rush/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -181,6 +189,39 @@ export default class RushConfiguration {
     return tempNamesByProject;
   }
 
+  private static _validateCommonRushConfigFolder(commonRushConfigFolder: string): void {
+    if (!fsx.existsSync(commonRushConfigFolder)) {
+      console.log(`Creating folder: ${commonRushConfigFolder}`);
+      fsx.mkdirsSync(commonRushConfigFolder);
+      return;
+    }
+
+    const filenames: string[] = fsx.readdirSync(commonRushConfigFolder);
+    for (const filename of filenames) {
+      const resolvedFilename: string = path.resolve(commonRushConfigFolder, filename);
+
+      // Ignore things that aren't actual files
+      const stat: fsx.Stats = fsx.statSync(resolvedFilename);
+      if (!stat.isFile() && !stat.isSymbolicLink) {
+        continue;
+      }
+
+      // Ignore harmless file extensions
+      const fileExtension: string = path.extname(filename);
+      if (['.bak', '.md', '.old', '.disabled'].indexOf(fileExtension) >= 0) {
```

Comment:
Fixed

---
In reply to: [115866578](https://github.com/Microsoft/web-build-tools/pull/176#discussion_r115866578) [](ancestors = 115866578)